scripts/xe-backup-metadata

```diff
@@ -134,11 +156,25 @@ function cleanup {
    fi
 }
 
+# if we can't validate the UUID of the VDI, prompt the user
+if [ -n "${vdi_uuid}" ]; then
+    if ! validate_vdi_uuid "${sr_uuid}" "${vdi_uuid}"; then
+        local version="8.4"
+        echo "Backup VDI $vdi was created prior to XenServer $version."
+        echo "Make sure you are using a VDI that you created as we"
+        echo "can't validate it without mounting it."
+        read -p "Continue? [Y/N]" -n 1 -r; echo
+        if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
+    fi
+fi
+
 echo Using SR: ${sr_name}
```

Comment:
In xe-backup-metadata line 171:
echo Using SR: ${sr_name}
               ^--------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
echo Using SR: "${sr_name}"